src/serialbox-python/compare/compare.py

```diff
@@ -0,0 +1,404 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+##===-----------------------------------------------------------------------------*- Python -*-===##
+##
+##                                   S E R I A L B O X
+##
+## This file is distributed under terms of BSD license. 
+## See LICENSE.txt for more information.
+##
+##===------------------------------------------------------------------------------------------===##
+##
+## Compares Serialbox fields in FILE_1 and FILE_2.
+## FILE_1 and FILE_2 can be .dat or .json files, but need to be of the same type. When they are
+## .json files, all fields are compared. When they are .dat files, the field name has to be the
+## last part of the file name, separated by '_'.
+##
+##===------------------------------------------------------------------------------------------===##
+
+from __future__ import print_function
+
+from sys import exit, stderr, version_info
+
+# Check Python version
+if version_info < (3, 4):
+    from platform import python_version
+
+    print("compare: error: compare requires atleast python 3.4 (detected %s)" % python_version(),
```

Comment:
Typo: at least with a space between at and least

Reply:
done